pages/task/schema/confirm.js

```diff
@@ -19,6 +19,9 @@ module.exports = (task, chosenStatus) => {
     if (action === 'grant-ra') {
       return false;
     }
+    if (chosenStatus === 'returned-to-applicant') {
+      return false;
+    }
```

Comment:
This stops establishment admins being asked about AWERB when returning to the applicant